### PR TITLE
feat: added a flag to disable synchronization

### PR DESCRIPTION
**Pull request by agusaldasoro**
This PR fixes https://github.com/decentraland/catalyst/issues/192

### CODE REVIEW

File: content/src/Server.ts [hunk 1]
```diff
@@ -47,12 +50,18 @@ export class Server {
...
     this.garbageCollectionManager = env.getBean(Bean.GARBAGE_COLLECTION_MANAGER)
-    this.synchronizationManager = env.getBean(Bean.SYNCHRONIZATION_MANAGER)
     this.snapshotManager = env.getBean(Bean.SNAPSHOT_MANAGER)
     this.service = env.getBean(Bean.SERVICE)
     this.migrationManager = env.getBean(Bean.MIGRATION_MANAGER)
     this.repository = env.getBean(Bean.REPOSITORY)
 
+    if (env.getConfig(EnvironmentConfig.DISABLE_SYNCHRONIZATION)) {
+      Server.LOGGER.warn(`Cluster synchronization has been disabled.`)
+      this.synchronizationManager = new DisabledSynchronizationManager()
```
Comment (agusaldasoro) [inline content/src/Server.ts:60]: I could just modified the bean for that  case, but it seem more clear just to have an env var 🤔

File: content/test/integration/E2ETestEnvironment.ts [hunk 2]
```diff
@@ -217,14 +223,18 @@ class PostgresWaitStrategy extends LogWaitStrategy {
...
 
+export function loadTestEnvironmentWithoutSynchronization(): E2ETestEnvironment {
+  return loadTestEnvironment(new Map([[EnvironmentConfig.DISABLE_SYNCHRONIZATION, true]]))
+}
+
 /**
  * This is an easy way to load a test environment into a test suite
  */
-export function loadTestEnvironment(): E2ETestEnvironment {
+export function loadTestEnvironment(overrideConfigs?: Map<EnvironmentConfig, any>): E2ETestEnvironment {
```
Comment (nchamo) [inline content/test/integration/E2ETestEnvironment.ts:233]: Nice!

File: content/test/integration/E2ETestEnvironment.ts [hunk 3]
```diff
@@ -217,14 +223,18 @@ class PostgresWaitStrategy extends LogWaitStrategy {
   }
 }
 
+export function loadTestEnvironmentWithoutSynchronization(): E2ETestEnvironment {
```
Comment (nchamo) [inline content/test/integration/E2ETestEnvironment.ts:226]: Let's make it simple by calling it something like `loadStandaloneTestEnvironment`

File: content/src/Server.ts [hunk 4]
```diff
@@ -47,12 +50,18 @@ export class Server {
     const upload = multer({ dest: Server.UPLOADS_DIRECTORY, preservePath: true })
     const controller: Controller = env.getBean(Bean.CONTROLLER)
     this.garbageCollectionManager = env.getBean(Bean.GARBAGE_COLLECTION_MANAGER)
-    this.synchronizationManager = env.getBean(Bean.SYNCHRONIZATION_MANAGER)
     this.snapshotManager = env.getBean(Bean.SNAPSHOT_MANAGER)
     this.service = env.getBean(Bean.SERVICE)
     this.migrationManager = env.getBean(Bean.MIGRATION_MANAGER)
     this.repository = env.getBean(Bean.REPOSITORY)
 
+    if (env.getConfig(EnvironmentConfig.DISABLE_SYNCHRONIZATION)) {
```
Comment (nchamo) [inline content/src/Server.ts:58]: Let's move this complexity inside the `SynchronizationManager`. We don't want to have all domain config & logic be here in one place.

We can move this inside the `SynchronizationManager#start`

**Review by nchamo — CHANGES_REQUESTED** [review]

**Review by nchamo — APPROVED** [review]
Great work!

**Pull request merged**